alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts

```diff
@@ -1,14 +1,5 @@
-import { AutoMap } from '@automapper/classes';
-import { Column, Entity } from 'typeorm';
+import { Entity } from 'typeorm';
 import { BaseCodeEntity } from '../../../common/entities/base.code.entity';
 
 @Entity()
-export class ApplicationDecisionMaker extends BaseCodeEntity {
-  @AutoMap()
-  @Column()
-  label: string;
-
-  //TODO: Why is this not getting mapped from the inherited class?
-  @AutoMap()
```

Comment:
Did you figure out what was the issue?

Reply:
No, even odder is that it appears to only be an issue with ApplicationStatus. All the other code entities are returning with description, status does not whether this is added or removed.

Reply:
Not sure why I though that, it appears to not work for any of them with or without this.

Reply:
Found the actual issue, description was missing a type on the entity. Fixed.